Misha/Numbers/find_cost_of_tile.txt

```diff
@@ -0,0 +1,2 @@
+100 90 50 50
```

Comment:
Move find_cost_of_tile.txt to input folder
Misha/Numbers/input_data/find_cost_of_tile.txt
Misha/Numbers/output_data/find_cost_of_tile_output.txt